### PR TITLE
ci: kbs: Enable CDH tests for TDX

### PR DESCRIPTION
**Pull request by fidencio**
Let's enable the CDH tests for TDX.

### CODE REVIEW

**Comment by fidencio** [comment]
**PLEASE, DO NOT YET ADD LABELS TO RUN THE TESTS**, as we're depending on the change to be done on the CI machine.

Once the change is made, I will add the labels to run the CI, and once the CI passes, I will convert this PR "Ready to review".

**Review by dborquez — DISMISSED** [review]
lgtm, thank you @fidencio

File: tests/integration/kubernetes/confidential_kbs.sh [hunk 1]
```diff
@@ -248,10 +248,10 @@ function kbs_k8s_deploy() {
 
 	echo "::group::Deploy the KBS"
 	if [ "${KATA_HYPERVISOR}" = "qemu-tdx" ]; then
-		echo "Setting up custom PCCS for TDX"
+		echo "Setting up custom Collateral Service for TDX"
 		cat <<- EOF > "${COCO_KBS_DIR}/config/kubernetes/custom_pccs/sgx_default_qcnl.conf"
 {
- "pccs_url": "https://localhost:8081/sgx/certification/v4/",
+ "collateral_service": "https://api.trustedservices.intel.com/sgx/certification/v4/",
```
Comment (wainersm) [inline tests/integration/kubernetes/confidential_kbs.sh:254]: Hi @fidencio !

I'd like to learn more about that part of Intel TDX/SGX attestation; is there a intro public link that you could share with me?

**Review by wainersm — APPROVED** [review]
I'm glad to see the CDH tests infra is ready for TDX!

Thanks @fidencio !

File: tests/integration/kubernetes/k8s-confidential-attestation.bats [hunk 2]
```diff
@@ -15,7 +15,6 @@ export KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 export AA_KBC="${AA_KBC:-cc_kbc}"
 
 setup() {
-	[ "${KATA_HYPERVISOR}" = "qemu-tdx" ] && skip "Test not ready yet for ${KATA_HYPERVISOR}"
```
Comment (wainersm) [inline tests/integration/kubernetes/k8s-confidential-attestation.bats:18]: \o/

**Review by wainersm — APPROVED** [review]
Great! thanks @fidencio !

File: tests/integration/kubernetes/confidential_kbs.sh (hunk 1 above)
Comment (fidencio) [reply tests/integration/kubernetes/confidential_kbs.sh]: https://download.01.org/intel-sgx/sgx-dcap/1.21/linux/docs/ contains a whole lot of documents about DCAP, which is what we use for attestation.

**Review by dborquez — APPROVED** [review]
Lgtm, thank you @fidencio

**Pull request merged**